Libraries/vendor/react/browser/eventPlugins/PanResponder.js

```diff
@@ -25,7 +25,24 @@ var currentCentroidY = TouchHistoryMath.currentCentroidY;
  * It provides a predictable wrapper of the responder handlers provided by the
  * [gesture responder system](/react-native/docs/gesture-responder-system.html).
  * For each handler, it provides a new `gestureState` object alongside the
- * normal event.
+ * native event ‘evt’ object:
```

Comment:
Remove the text `evt`. It's just a variable name.

`native event ‘evt’ object` -> `native event object`